### PR TITLE
refractor!: removing hardcoded ids from the lockdown command

### PR DESCRIPTION
**Pull request by DEV-DIBSTER**
Going to try to create a format for removing all hardcoded IDs within the bot. Also trying to limit the influence of global configurations.

Changes to the configuration file as followed:

image
image

### CODE REVIEW

File: example-config.json [hunk 1]
```diff
@@ -1,135 +1,207 @@
...
-    "reactionRoles": {
-      "898041838701781013": {
-        "918288000071381002": {
-          "📣": "898041778471592027",
-          "♻️": "795964657239326742"
+            "member": "",
+            "client": "",
+            "bot": "",
+            "staff": "",
+            "mute": ""
```
Comment (wdhdev) [inline example-config.json:58]: We don't use this role anymore

File: example-config.json [hunk 2]
```diff
@@ -1,135 +1,207 @@
...
+        "inviterewmsg": "",
+        "invites": {
+            "5": "",
+            "10": "",
+            "25": "",
+            "50": "",
+            "100": "",
+            "150": "",
+            "200": ""
         },
```
Comment (wdhdev) [inline example-config.json:70]: We don't use these anymore

File: example-config.json [hunk 3]
```diff
@@ -1,135 +1,207 @@
...
+                    "🖲️": "",
+                    "✨": ""
+                }
+            },
+            "754441222424363088": {
+                "770046936957845514": {
+                    "🔪": ""
+                }
+            }
+        },
```
Comment (wdhdev) [inline example-config.json:96]: We don't have reaction roles anymore

File: example-config.json [hunk 4]
```diff
@@ -1,135 +1,207 @@
...
+            "Helper": "",
+            "Staff": "",
+            "NewTicket": "",
+            "BetaTester": "",
+            "Booster": "",
+            "Donator": "",
+            "Client": "",
+            "Member": "",
+            "Bot": "",
+            "Muted": "",
```
Comment (wdhdev) [inline example-config.json:113]: Not needed

File: example-config.json [hunk 5]
```diff
@@ -1,135 +1,207 @@
...
+            "TicketBan": "",
+            "Announcements": "",
+            "Changelogs": "",
+            "Giveaways": "",
+            "Polls": "",
+            "GameNights": "",
+            "MovieNights": "",
+            "StaffUpdates": "",
+            "Others": ""
```
Comment (wdhdev) [inline example-config.json:122]: Not needed

File: example-config.json [hunk 6]
```diff
@@ -1,135 +1,207 @@
...
+            "Staff": "",
+            "NewTicket": "",
+            "BetaTester": "",
+            "Booster": "",
+            "Donator": "",
+            "Client": "",
+            "Member": "",
+            "Bot": "",
+            "Muted": "",
+            "TicketBan": "",
```
Comment (wdhdev) [inline example-config.json:114]: Handled by beta bot

File: example-config.json [hunk 7]
```diff
@@ -1,135 +1,207 @@
...
+        "Password": "",
+        "From": ""
+    },
+    "Email2": {
+        "Host": "",
+        "Port": "",
+        "User": "",
+        "Password": "",
+        "From": ""
+    },
```
Comment (wdhdev) [inline example-config.json:155]: Why is there two email configs?

File: example-config.json [hunk 8]
```diff
@@ -1,135 +1,207 @@
...
+    "PrivPterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
+    "DanPterodactyl": {
+        "hosturl": "",
+        "apikeyclient": ""
+    },
```
Comment (wdhdev) [inline example-config.json:141]: Where is this needed?

File: example-config.json [hunk 9]
```diff
@@ -1,135 +1,207 @@
...
+        "pteropublic": "",
+        "vpnapi": ""
+    },
+    "BannedNames": [
+        "",
+        "",
+        "",
+        "",
+        ""
+    ],
```
Comment (wdhdev) [inline example-config.json:194]: What's this?

File: example-config.json [hunk 10]
```diff
@@ -1,135 +1,207 @@
...
+    "Enabled": {
+        "NodeStats": "",
+        "Welcome": "",
```
Comment (wdhdev) [inline example-config.json:197]: Handled by beta bot

File: example-config.json [hunk 11]
```diff
@@ -1,135 +1,207 @@
...
+    "Pterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
+    "PrivPterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
```
Comment (wdhdev) [inline example-config.json:137]: Isn't this on an entirely different bot?

File: example-config.json [hunk 12]
```diff
@@ -1,135 +1,207 @@
...
+    "Pterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
+    "PrivPterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
```
Comment (wdhdev) [inline example-config.json:136]: Why is a client key needed?

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by ninsaccplayzyt** [comment]
removing hardcoded ids goes crazy

File: example-config.json (hunk 6 above)
Comment (DEV-DIBSTER) [reply example-config.json:114]: Will keep for clarity.

**Comment by DEV-DIBSTER** [comment]
> removing hardcoded ids goes crazy

Getting rid of them all. Slowly but surely.

File: example-config.json (hunk 11 above)
Comment (DEV-DIBSTER) [reply example-config.json:137]: Before stuff.

File: example-config.json (hunk 12 above)
Comment (DEV-DIBSTER) [reply example-config.json:136]: Before stuff.

File: example-config.json (hunk 10 above)
Comment (DEV-DIBSTER) [reply example-config.json:197]: No, NodeStats is still used in the code.

File: example-config.json (hunk 9 above)
Comment (DEV-DIBSTER) [reply example-config.json:194]: Banned nicknames.

File: example-config.json (hunk 8 above)
Comment (DEV-DIBSTER) [reply example-config.json:141]: Before stuff.

File: example-config.json (hunk 7 above)
Comment (DEV-DIBSTER) [reply example-config.json:155]: Always has been.

**Pull request merged**